DFC.App.Triagetool/Views/Pages/Body.cshtml

```diff
@@ -1,74 +1,63 @@
+                                    } 
+                                    else
+                                    {
+                                        <input  typeof= "checkbox" class="govuk-checkboxes__input" name="filterAdviceGroupOptions" id="@fac.ContentItemId" type="checkbox" value="@fac.ContentItemId" />
 
+                                    }
+                                                        <label class="govuk-label govuk-checkboxes__label" for="@fac.ContentItemId">
+                                                @fac.Title
+                                            </label>
+                                        </div>
+                                </div>
                             </div>
-                        </div>
-                    </div>
-                    <noscript>
-                        <button data-module="govuk-button" class="govuk-button" type="submit">Apply filters</button>
-                    </noscript>
-                    <input id="SelectedOption" name="triage-select" type="hidden" value="@Model.Title">
-                    <p class="clear-filters"><a id="clearFilters" href="/triagetool/@Model.Title">Clear filters</a></p>
-                    <div class="triage-tool-sidebar">
-                        @Html.Raw(Model.SharedContent)
-                    </div>
-                    <select class="govuk-select ncs-triage-select" id="pocSelect" name="poc-select">
-                        <option value="union">original</option>
-                        <option value="intersection">option 3</option>
-                        <option value="filters" selected>option 4</option>
-                        <option value="radio">option 5</option>
-                        <option value="disable">option 6</option>
-                    </select>
-                </div>
-            </div>
-            <div class="govuk-grid-column-two-thirds">
-                <p class="govuk-body">Showing <span class="fac-result-count triage-result-count" id="totalArticles">@Model.Pages.Count @(Model.Pages.Count == 1 ? "suggestion": "suggestions")</span> for "<span class="govuk-!-font-weight-bold" id="primaryFiltersSelectedValue">@Model.Title</span>"</p>
-                <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-6 govuk-!-margin-top-2">
-                <div class="triage-opt2 results" id="allArticles">
-                    <div class="results-block">
-                        <div class="inner-block">
-                            @(await Html.PartialAsync("TriageToolPartialViews/TriageToolPage", Model))
-                        </div>
+
+                            }
+                        }
+                     </div>
+                     <div>
+                <button type="submit" class="govuk-button dfc-app-triage-btn ncs-button__primary govuk-button--start jscript-disabled-btn" id="applyFiltersJs" name="FilterAction" value="ApplyFilters">Apply Filters</button>
```

Comment:
Duplicated HTML buttons?

Reply:
They aren't duplicate. one for JS script disabled